Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.42858&attempt_id=b3653256-d13b-4461-85c1-10fb07f42abc)

Attempted: rspamd

<details><summary>Partial log (click to expand)</summary><p>

```
/build/source/src/libserver/fuzzy_backend_redis.c:1362:15: warning: 'cmd' may be used uninitialized in this function [-Wmaybe-uninitialized]
  session->cmd = cmd;
  ~~~~~~~~~~~~~^~~~~
make[1]: *** [CMakeFiles/Makefile2:1545: src/CMakeFiles/rspamd-server.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 85%] Linking C shared library libTH.so
[ 85%] Built target TH
make: *** [Makefile:130: all] Error 2
builder for '/nix/store/i984m3r9k5bxpv8ms69x5a5gilzf1na3-rspamd-1.7.4.drv' failed with exit code 2
error: build of '/nix/store/i984m3r9k5bxpv8ms69x5a5gilzf1na3-rspamd-1.7.4.drv' failed
```
</p></details>

--- a/pkgs/servers/mail/rspamd/default.nix
+++ b/pkgs/servers/mail/rspamd/default.nix
@@ -6,22 +6,17 @@ in
 
 stdenv.mkDerivation rec {
   name = "rspamd-${version}";
-  version = "1.6.6";
+  version = "1.7.3";
 
   src = fetchFromGitHub {
     owner = "vstakhov";
     repo = "rspamd";
     rev = version;
-    sha256 = "04jqrki7rlxywdig264kavy1h5882rspi2drkbdzrk35jjq8rh3h";
+    sha256 = "1gb4zg8i1nj337f65s434h299ad19c0d7jyawb2glvv3n4cshm97";
   };
 
   nativeBuildInputs = [ cmake pkgconfig perl ];
   buildInputs = [ glib gmime libevent libmagic luajit openssl pcre sqlite ragel icu libfann];
-
-  postPatch = ''
-    substituteInPlace conf/common.conf --replace "\$CONFDIR/rspamd.conf.local" "/etc/rspamd/rspamd.conf.local"
-    substituteInPlace conf/common.conf --replace "\$CONFDIR/rspamd.conf.local.override" "/etc/rspamd/rspamd.conf.local.override"
-  '';
 
   cmakeFlags = ''
     -DDEBIAN_BUILD=ON

--- a/pkgs/servers/mail/rspamd/default.nix
+++ b/pkgs/servers/mail/rspamd/default.nix
@@ -16,14 +16,15 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ cmake pkgconfig perl ];
-  buildInputs = [ glib gmime libevent libmagic luajit openssl pcre sqlite ragel icu libfann];
+  buildInputs = [ glib gmime libevent libmagic luajit openssl pcre sqlite ragel icu libfann ];
 
-  cmakeFlags = ''
-    -DDEBIAN_BUILD=ON
-    -DRUNDIR=/var/run/rspamd
-    -DDBDIR=/var/lib/rspamd
-    -DLOGDIR=/var/log/rspamd
-  '';
+  cmakeFlags = [
+    "-DDEBIAN_BUILD=ON"
+    "-DRUNDIR=/var/run/rspamd"
+    "-DDBDIR=/var/lib/rspamd"
+    "-DLOGDIR=/var/log/rspamd"
+    "-DLOCAL_CONFDIR=/etc/rspamd"
+  ];
 
   meta = with stdenv.lib; {
     homepage = https://github.com/vstakhov/rspamd;

--- a/pkgs/servers/mail/rspamd/default.nix
+++ b/pkgs/servers/mail/rspamd/default.nix
@@ -6,13 +6,13 @@ in
 
 stdenv.mkDerivation rec {
   name = "rspamd-${version}";
-  version = "1.7.3";
+  version = "1.7.4";
 
   src = fetchFromGitHub {
     owner = "vstakhov";
     repo = "rspamd";
     rev = version;
-    sha256 = "1gb4zg8i1nj337f65s434h299ad19c0d7jyawb2glvv3n4cshm97";
+    sha256 = "1iba6mpha1ikybn9qnvgxzh6pjw5yj5aipamd586rfb0j9lbwsd5";
   };
 
   nativeBuildInputs = [ cmake pkgconfig perl ];

